infra/stages/production/terraform.tfvars

```diff
@@ -0,0 +1,19 @@
+aws_account_id = "627002765486"
+
+aws_region = "eu-west-1"
+
+aws_profile = "thalia"
+
+aws_tags = {
+  "Category"    = "thadmin",
+  "Owner"       = "technicie",
+  "Environment" = "production",
+  "Terraform"   = true
+}
+
+prefix = "thadmin"
+
+stage = "production"
+
+domain_name = "sales.thalia.nu"
```

Comment:
I prefer thadmin.thalia.nu as people will also call it 'thadmin'

Reply:
Done